alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md

```diff
@@ -0,0 +1,121 @@
+# PrometheusScrapeBodySizeLimitHit
+
+## Meaning
+
+The alert `PrometheusScrapeBodySizeLimitHit` is triggered when at least one of
+Prometheus' scrape targets replies with a response body larger than `body_size_limit`.
+
+## Impact
+
+Metrics coming from targets responding with a body exceeding the configured
+size limit aren't ingested by Prometheus. The targets will be considered as being
+down and they will have their `up` metric set to 0 which may also trigger the
+`TargetDown` alert.
+
+## Diagnosis
+
+First we need check the value of the configured bodysize limit.
+This value is located either in the configmap `cluster-monitoring-config` in the
+namespace `openshift-monitoring`, or in the Prometheus configuration
+file, which is saved in the secret `prometheus-k8s` in the namespace `openshift-monitoring`.
+Search for the field `body_size_limit`.
+
+- To check the value in configmap, use this command:
+```bash
+kubectl get cm -n openshift-monitoring cluster-monitoring-config -o yaml  | grep enforcedBodySizeLimit
+```
+
+- To check the valude in Prometheus configuration file, use this command instead:
+```bash
+kubectl get secret -n openshift-monitoring prometheus-k8s -o yaml | grep prometheus.yaml.gz: | sed -e 's/^[ ]*prometheus.yaml.gz: *//' | base64 -d | gunzip -c
```

Comment:
I still believe that it's too brittle and not really necessary to check the Prometheus configuration.
Checking the value configured in the Prometheus object is good enough from my POV (if the limit isn't propagated to the Prometheus config file, we have a bigger problem).

```
oc get -n openshift-monitoring prometheus k8s -o jsonpath='{.spec.enforcedBodySizeLimit}'
```